CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Updated loki-write hpa in the values so that it also takes into account memory for scaling the pods.
```

Comment:
Why only on the write ?

Reply:
Because the issue we had was with the write component. Since we didn't have the same kind of issue with other components up until now, I think we should do things incrementally and add the memory for other components when/if we encounter the same situation.

But I'm also ok to add this to all components if you'd prefer :shrug:

Reply:
I would probably have it everywhere yes

Reply:
IMO I'd like to add the memory on other hpa and not waiting for new issues :-)

Reply:
Works for me :)